library/Ivoz/Provider/Domain/Service/Invoice/CreateByScheduler.php

```diff
@@ -116,7 +118,15 @@ private function createInvoice(InvoiceSchedulerInterface $scheduler)
             )
             ->setSchedulerId(
                 $scheduler->getId()
+            )
+            ->setTotal(0);
+
+        if (!is_null($schedulerNumberSequence)) {
```

Comment:
This is already being done in `Ivoz\Provider\Domain\Service\Invoice\SetInvoiceNumber`

Reply:
But if I remove that check, php-stan will dispatch an exception indicating unable to access methods of null values.

Reply:
I mean that you don't have to call `$invoiceDto->setNumber(` at all